lib/exfile_imagemagick/sys_runner.ex

```diff
@@ -1,9 +1,20 @@
 defmodule ExfileImagemagick.SysRunner do
   alias ExfileImagemagick.Server
 
-  def cmd(command, args) do
+  @image_processor Application.get_env(:exfile_imagemagick, :image_processor, :imagemagick)
```

Comment:
If you move this to `ExfileImagemagick.Server`, the code could be cleaned up a bit.